lua/nvim-tree/actions/run-command.lua

```diff
@@ -0,0 +1,17 @@
+local M = {}
+
+function M.run_file_command(node)
+  vim.api.nvim_input(": " .. node.absolute_path .. "<Home>")
+end
+
+function M.run_directory_command(node)
```

Comment:
I'm not sure if we need a dedicated `run_directory_command`. It seems to correspond to `Pb` (`P` to go to the parent node and then `b`

Removing it would lower the complexity of this function a bit.

I'm not the maintainer, so don't take my word for it :smile: I'm just stating my point of view

Reply:
Yepp, I removed it :) So we end with a function containing one line of code :-D

Reply:
Nice, simple is beautiful :smile: